900M/gui.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * gui.js
+ **/
+
+function buildGui(options, callback) {
```

Comment:
'buildGui' is defined but never used.<br>'callback' is defined but never used.<br>'options' is defined but never used.